### PR TITLE
Copy-edit pull request

### PR DESCRIPTION
**Pull request by rajdinh**
Please note this copy-edit is ONLY for the 800-63-3 document and does not include A, B, or C.

Copy-edit done based on list of priorities sent over - so first acronym use, consistency in lists, headers, spacing and more. Additional comments (mostly about a few references) have also been emailed.

### CODE REVIEW

**Comment by rajdinh** [comment]
Please note this copy-edit is ONLY for the 800-63-3 document and does not include A, B, or C.

File: sp800-63-3/sec1_2_introduction.md [hunk 1]
```diff
@@ -30,34 +29,33 @@ The components of identity assurance detailed in these guidelines are as follows
...
 
 As such, SP 800-63 is organized as a suite of documents as follows:
 
 - SP 800-63-3 *Digital Identity Guidelines* - Provides an overview of general identity frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _This document contains only informative material._
 
-- SP 800-63A *Enrollment and Identity Proofing* - Addresses how applicants can prove their identities and become enrolled as valid subjects within an identity system. It provides guidelines for processes by which applicants can both proof and enroll at one of three different levels of risk mitigation in both remote and physically-present scenarios. _This document contains both normative and informative material._
+- SP 800-63A *Enrollment and Identity Proofing* - Addresses how applicants can prove their identities and become enrolled as valid subjects within an identity system. It provides a guideline for processes by which applicants can both proof and enroll at one of three different levels of risk mitigation in both remote and physically-present scenarios. _This document contains both normative and informative material._
```
Comment (paul-grassi) [inline sp800-63-3/sec1_2_introduction.md]: a guideline -> requirements

File: sp800-63-3/sec1_2_introduction.md [hunk 2]
```diff
@@ -67,49 +65,49 @@ their authentication assurance requirements:
     this document suite.
 
 4.  *Validate that the implemented system has met the required assurance
-    level* – As some implementations may create or compound particular
+    level* - As some implementations may create or compound particular
     risks, agencies should conduct a final validation to confirm that
     the system achieves the required assurance level for the
     user-to-agency process. NIST SP 800-53A [[SP 800-53A]](#SP800-53A)
-    provides guidelines for the assessment of the implemented system
+    provides a guideline for the assessment of the implemented system
```
Comment (paul-grassi) [inline sp800-63-3/sec1_2_introduction.md]: a guideline -> requirements/recommendations.

File: sp800-63-3/sec1_2_introduction.md [hunk 3]
```diff
@@ -30,34 +29,33 @@ The components of identity assurance detailed in these guidelines are as follows
...
 
-- SP 800-63C *Federation and Assertions* - Provides guidelines on the use of federated identity architectures and assertions to convey the results of authentication processes and relevant identity information to an agency application. In addition, these guidelines offer privacy enhancing techniques to share information about a valid, authenticated subject, as well as describing methods that allow for strong multifactor authentication while the subject remains pseudonymous to the digital service. _This document contains both normative and informative material._
+- SP 800-63C *Federation and Assertions* - Provides a guideline on the use of federated identity architectures and assertions to convey the results of authentication processes and relevant identity information to an agency application. In addition, this guideline offers privacy enhancing techniques to share information about a valid, authenticated subject, as well as describing methods that allow for strong multi-factor authentication (MFA) while the subject remains pseudonymous to the digital service. _This document contains both normative and informative material._
```
Comment (paul-grassi) [inline sp800-63-3/sec1_2_introduction.md]: a guideline -> requirements

File: sp800-63-3/sec1_2_introduction.md [hunk 4]
```diff
@@ -67,49 +65,49 @@ their authentication assurance requirements:
...
     part of a security authorization process as described in NIST SP
     800-37, Revision 1 [[SP 800-37]](#SP800-37).
 
 5.  *Periodically reassess the system to determine
-    technology refresh requirements* – The agency shall periodically
+    technology refresh requirements* - The agency shall periodically
     reassess the system to ensure that the identity
     authentication requirements continue to be satisfied. NIST SP
-    800-37, Revision 1 [[SP 800-37]](#SP800-37) provides guidelines on
+    800-37, Revision 1 [[SP 800-37]](#SP800-37) provides a guideline on
```
Comment (paul-grassi) [inline sp800-63-3/sec1_2_introduction.md]: guideline -> requirements/recommendation

File: sp800-63-3/sec1_2_introduction.md [hunk 5]
```diff
@@ -67,49 +65,49 @@ their authentication assurance requirements:
...
     the frequency, depth and breadth of periodic reassessments. As with
     the initial validation process, agencies should follow the
-    assessment guidelines specified in SP 800-53A [[SP
+    assessment guideline specified in SP 800-53A [[SP
```
Comment (paul-grassi) [inline sp800-63-3/sec1_2_introduction.md]: guideline -> requirements/recommendation

**Review by paul-grassi — CHANGES_REQUESTED** [review]

**Review by paul-grassi — APPROVED** [review]

**Pull request merged**